javascript/2-functions/hw-2/scripts.js

```diff
@@ -1 +1,17 @@
 // Your JS code is here
+function biggerNum(num1, num2) {
+  if (typeof num1 !== "number" || typeof num2 !== "number") {
+    return "not a number";
+  } else if (num1 === num2) {
+    return "numbers are equal";
+  } else if (num2 > num1) {
+    return num2;
+  } else {
+    return "Error";
+  }
+}
+
+biggerNum(3, 1); // 3
```

Comment:
first test did not pass. please try to fix it.

Reply:
Code fixed pls review